src/main/scala/edu/berkeley/ce/rockslicing/BlockVTK.scala

```diff
@@ -0,0 +1,197 @@
+package edu.berkeley.ce.rockslicing
+
+import breeze.linalg
+import breeze.linalg.{DenseVector, DenseMatrix}
+
+object BlockVTK {
+
+  private val EPSILON = 1.0e-6
+
+  /**
+   * Calculates the rotation matrix to rotate the input plane (specified by its normal)
```

Comment:
The convention that we've been using for multiline documentation comments looks like this:
```Scala
/**
  * foo
  * bar
  */
```

The asterisks all line up with the second asterisk on the first line. So we're consistent, I think we should stick with that.